packages/marko-web-search/config/query-params.js

```diff
@@ -66,6 +66,12 @@ class MarkoWebSearchQueryParamConfig {
         defaultValue: () => [],
         toInput: toIntArrayInput,
         fromInput: fromArrayInput,
+      })
+      .add('sortBy', {
+        type: Array,
+        defaultValue: () => [],
+        toInput: toStringArrayInput,
+        fromInput: fromArrayInput,
       });
```

Comment:
At top of file, add: `const sortFieldSet = new Set('PUBLISHED', 'SCORE')`;

Add default: 
`defaultValue: 'PUBLISHED'`

Add validator:

```js
validator: v => sortFieldSet.has(v),
```